colors/build.js

```diff
@@ -2,29 +2,43 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
-  for (var variant in element) {
+function createColor(color, element, prefix) {
+  const rv = [];
+
+  for (const variant in element) {
     if (element.hasOwnProperty(variant)) {
-      var value = element[variant];
-      rv.push(`  --${color}-${variant}: ${value};\n`);
+      const value = element[variant];
+      rv.push(`${prefix}${color}-${variant}: ${value};\n`);
```

Comment:
We could also inline `value` here, to save the extra variable…  (I won't insist on that, though, so only change it if you feel like it, or if there are other changes to make.  🙂)